tabular/src/autogluon/tabular/predictor/predictor.py

```diff
@@ -158,18 +166,22 @@ def __init__(
             eval_metric=None,
             path=None,
             verbosity=2,
+            sample_weight=None,
+            weight_evaluation=False,
             **kwargs
     ):
         self.verbosity = verbosity
         set_logger_verbosity(self.verbosity, logger=logger)
+        self.sample_weight = sample_weight  # TODO: add support for sample_weight= 'auto', 'balanced'
```

Comment:
sample_weight and weight_evaluation can both be properties that link to self._learner.sample_weight, self._learner.weight_evaluation. Currently it is defined twice which isn't ideal.

Reply:
not exactly sure what to do here as I didn't see any examples, so just marked as TODO

Reply:
Sure, it isn't urgent. Just a minor detail that I can cleanup afterwards